torch/_inductor/codecache.py

```diff
@@ -1209,6 +1209,23 @@ def post_compile(
 
         return compiled_graph
 
+    @staticmethod
+    def __should_save_graph(compiled_graph: CompiledFxGraph):
+        # Do not use any heuristics in unit tests
+        if torch._utils_internal.is_fb_unit_test():
```

Comment:
see also `IS_CI`

Reply:
is that something ok to read in codecache? It looks like something only meant to be used in unit tests

Reply:
Since IS_IC is OSS only, we dont need to add it since this was to guard against JK changes.

Reply:
Really would prefer not adding skew between internal tests and oss tests - that leads to all sorts of problems and is hard to debug

Reply:
OSS unit tests will currently save because `minimum_time=None` and `justknobs_getval_int` will return 0 in OSS? Maybe future proof if we change the default to something other than None by returning True here if `is_fb_unit_test()` or is OSS test? I think that's what Elias is saying?

Reply:
Do the OSS tests have a backend to save to? (I'm surprised replacing is_fbcode with IS_CI has any impact on OSS tests).

Reply:
we have a redis implementation we generally mock out but there's also a local cache